packages/application-shell/src/components/application-shell/application-shell.spec.js

```diff
@@ -1142,7 +1142,7 @@ describe('when navbar menu items do not match given data fences', () => {
               value: 'usa',
               type: 'store',
               group: 'orders',
-              name: 'canManageOrders',
+              name: 'canViewOrders',
```

Comment:
Thanks! Was this related to the changes in this PR or was it wrong before?

Reply:
It related to the changes and was kind of right but also wrong. data fence permissions should infer view from manage. Just like with any other permission.